src/models/subscription.ts

```diff
@@ -190,6 +194,19 @@ export class Subscription extends Model {
 		return results[0] as SyncStatusCount[];
 	}
 
+	/*
+	 * Returns array or orgs that have connected repos
+	 */
+	static async getConnectedOrgs(jiraHost: string): Promise<RepoOwner[]> {
```

Comment:
Where is this being used?

Reply:
well...... its funny you should mention that.
I was originally leaving it there for the next step, for the search but its not going to work as I had hoped because we don't maintain repos :'(
So I shall remove it!